apps/storybook/src/ComboBox.stories.tsx

```diff
@@ -478,6 +479,81 @@ WithCustomMessageIcon.args = {
   ),
 };
 
+export const Loading: Story<Partial<ComboBoxProps<string>>> = (args) => {
+  const [options, setOptions] = React.useState<SelectOption<string>[]>([]);
+  const [selectedValue, setSelectedValue] = React.useState<string>();
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  const onChange = React.useCallback((value: string) => {
+    action(value ?? '')();
+    setSelectedValue(value);
+  }, []);
+
+  const emptyContent = React.useMemo(() => {
+    return isLoading ? (
+      <>
+        {new Array(6).fill(null).map((_, index) => {
+          // Randomize skeletons width
+          const normalizedIndex = (index % 2) + 1;
+          const randomValue = normalizedIndex * Math.random() * 60;
+          const boundedWidth = Math.min(Math.max(randomValue, 25), 60);
```

Comment:
Should we simplify this? I don't see the point of random widths

Reply:
It's just nicer for me 😀 but if concern is the amount of code, I can make it all the same width.

Reply:
yup readability is the concern. maybe we could reduce it a one liner and inline it like so:

```ts
contentWidth={Math.max(25, Math.random() * 60)} // randomize skeleton width
```

Reply:
Simplified. I don't know about visual testing as it might cause random failures because content changes after some time.